### PR TITLE
fix: update style and tooltips of asset widgets

### PR DESCRIPTION
**Pull request by yuda110**
### To Reviewers
- [ ] Skip (`style`, `chore`, `ci` ONLY)
- [x] Not that difficult

### Type of Change
- [ ] New feature
- [x] Bug fixes
- [ ] Feature improvement
- [ ] Refactor
- [x] Others (performance improvement, CI/CD, etc.)

### Affects to
- [ ] Packages
  - [ ] core-lib
  - [ ] mirinae
  - [ ] etc 

- [ ] Apps
  - [ ] storybook
  - [x] web

### Checklist
- Did you check the `lint` and `type`?
- Did you document the changes?
  - Changes in `mirinae` should be reflected in `storybook`.
- Did you test the app after the package changes?

### Description
SSIA

### CODE REVIEW

File: apps/web/src/common/composables/amcharts5/xy-chart-helper.ts [hunk 1]
```diff
@@ -255,6 +255,28 @@ export const setXYSharedTooltipTextByUsage = (chart: am5xy.XYChart, tooltip: am5
...
+            });
+        });
+        let _text = `Total: [bold; fontSize: 14px]${commaFormatter(totalValue)}[/]`;
+        seriesList.forEach((s) => {
+            const rate = Math.round((s.value / totalValue) * 100);
+            _text += `\n[${s.color}; fontSize: 10px]●[/] [fontSize: 14px;}]${s.name}:[/] [bold; fontSize: 14px]${commaFormatter(s.value)}[/] (${rate}%)`;
+        });
+        return _text;
+    });
+};
```
Comment (sulmoJ) [inline apps/web/src/common/composables/amcharts5/xy-chart-helper.ts:279]: praise: cool

**Review by sulmoJ — APPROVED** [review]
LGTM!

File: apps/web/src/services/dashboards/widgets/asset-widgets/total-failure-and-severity/TotalFailureAndSeverityWidget.vue [hunk 2]
```diff
@@ -379,6 +379,12 @@ defineExpose<WidgetExpose>({
                     @apply text-display-md;
                 }
                 .diff-wrapper {
+                    @screen mobile {
+                        .diff-text {
+                            display: none;
+                        }
+                    }
+
```
Comment (skdud4659) [inline apps/web/src/services/dashboards/widgets/asset-widgets/total-failure-and-severity/TotalFailureAndSeverityWidget.vue:387]: 🫶🏻

**Review by skdud4659 — APPROVED** [review]
The widget looks really challenging...
LGTM!!

**Pull request merged**